1_fetch/src/download_tifs_annual.R

```diff
@@ -32,13 +33,21 @@ download_tifs <- function(sb_id, filename, download_path, overwrite_file = T, ye
   # Find and delete years that are not needed - negate enables inverse matching
   if(!is.null(year)){
   years_collapsed <- year %>% lapply(function(x) paste0(x,'.tif')) %>% paste(collapse = '|')
-  files_del <- list.files(unzip_folder_path, full.names = T) %>% str_subset(years_collapsed, negate = T)
+  files_del <- list.files(unzip_folder_path, full.names = TRUE) %>% stringr::str_subset(years_collapsed, negate = T)
   lapply(files_del, file.remove)
   rm(path_to_downloaded_file, files_del)
   }
-
-   return(list.files(unzip_folder_path, full.names = T))
   
+  final_list <- list.files(unzip_folder_path, full.names = TRUE)
+
+  if(!is.null(name)){
+    
+    names(final_list) <- name
+    
+    }
+
+  print(final_list)
+  return(final_list)
```

Comment:
I am not sure that the labeling of the output is actually working here. Did it work for you?

Reply:
Oh, it's not naming them because `list.files` returns a character vector, not a list. I wonder if we need to name them? The file names already have the years in them.

Reply:
I'll remove it